arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,1020 @@
+		"EMMC_D0",
+		"EMMC_D1",
+		"EMMC_D2",
+		"EMMC_D3",
+		"SPI_4_CLK",
+		"SPI_4_MOSI",
+		"SPI_4_MISO",
+		"SPI_4_CS#",
+		"MAG_N_IRQ",
+		"FPGA_RESET#",
+		"SPI_3_FPGA_CS#",
+		"SPI_3_CLK",
+		"SPI_3_MOSI",
+		"SPI_3_MISO",
+		"PWM_5",
+		"UART6_TXD",
+		"UART6_RXD",
+		"ID5",
+		"GYRO_INT",
+		"6UL_FORCE_5V_ON",
+		"EN_EMMC_3.3V#",
+		"EN_YEL_LED#",
+		"EN_RED_LED#",
+		"EN_GRN_LED#",
+		"EN_BLU_LED",
+		"FRAM_SPI_CS#",
+		"SD_VSEL_1.8V",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpio5 {
+	gpio-line-names =
+		"POWER_FAIL",
+		"GPS_PPS_OUT",
+		"NA",
+		"GPIO_DVFS",
+		"NA",
+		"SILAB_C2_CLK",
+		"SILAB_C2_DATA",
+		"SILAB_C2_RESET",
+		"SPARE_4",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpc {
+	fsl,cpu_pupscr_sw2iso = <0x1>;
+	fsl,cpu_pupscr_sw = <0x0>;
+	fsl,cpu_pdnscr_iso2sw = <0x1>;
+	fsl,cpu_pdnscr_iso = <0x1>;
+	fsl,ldo-bypass = <0>; /* DCDC, ldo-enable */
+};
+
+&uart1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart1>;
+	status = "okay";
+};
+
+&uart2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart2>;
+	status = "okay";
+};
+
+&uart3 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart3>;
+	status = "okay";
+	uart-has-rtscts;
+};
+
+&uart4 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart4>;
+	status = "okay";
+	uart-has-rtscts;
+	rts-gpios = <&gpio1 19 GPIO_ACTIVE_HIGH>; /* EN_485 is the TX enable signal */
+	linux,rs485-enabled-at-boot-time;
+};
+
+&uart5 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart5>;
+	status = "okay";
+};
+
+
+&uart6 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart6>;
+	status = "okay";
+};
+
+&uart7 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart7>;
+	status = "okay";
+};
+
+&uart8 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_uart8>;
+	status = "okay";
+};
+
+&usdhc1 {
+	pinctrl-names = "default", "state_100mhz", "state_200mhz";
+	pinctrl-0 = <&pinctrl_usdhc1>;
+	pinctrl-1 = <&pinctrl_usdhc1_100mhz>;
+	pinctrl-2 = <&pinctrl_usdhc1_200mhz>;
+	max-frequency = <208000000>;
+	keep-power-in-suspend;
+	enable-sdio-wakeup;
+	disable-wp;
+	cap-sd-highspeed;
+	sd-uhs-sdr12;
+	sd-uhs-sdr25;
+	sd-uhs-sdr50;
+	sd-uhs-sdr104;
+	sd-uhs-ddr50;
+	broken-cd = <1>;
+	bus-width = <4>;
+	status = "okay";
+};
+
+&usdhc2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_usdhc2>;
+	no-1-8-v;
+	disable-wp;
+	broken-cd = <1>;
+	enable-sdio-wakeup;
+	bus-width = <4>;
+	status = "okay";
+};
+
+&i2c3 {
+	clock-frequency = <400000>;
+	pinctrl-names = "default", "gpio";
+	pinctrl-0 = <&pinctrl_i2c3>;
+	pinctrl-1 = <&pinctrl_i2c3_gpio>;
+	status = "okay";
+	scl-gpios = <&gpio3 6 GPIO_ACTIVE_HIGH>;
+	sda-gpios = <&gpio3 5 GPIO_ACTIVE_HIGH>;
+
+	gpio6: gpio@28 {
+		compatible = "technologic,ts4900-gpio";
+		reg = <0x28>;
+		#gpio-cells = <2>;
+		gpio-controller;
+		base = <160>;
```

Comment:
I believe `base` is deprecated. I think it should be removed, but I'm not 100% sure on that.

Reply:
Reopening this, base should be removed.

Reply:
Got it: "base" is no longer needed nor accepted.